packages/subgraph/manifest/templates/contracts/DisputableVoting.template.yaml

```diff
@@ -18,7 +18,9 @@
       - name: ConvictionVoting
         file: ./abis/ConvictionVoting.json
       - name: Agreement
-        file: ./abis/Agreement.json
+        file: ./abis/Agreement.json     
+      - name: ERC20
+        file: ./abis/ERC20.json
       - name: MiniMeToken
         file: ./abis/MiniMeToken.json
```

Comment:
```suggestion
      - name: MiniMeToken
         file: ./abis/MiniMeToken.json

Reply:
I think we don't need the ERC20 contract for the disputable voting.

Reply:
Ah no, in this is case needed as we use the ERC20 abi when loading the token at `loadTokenData`